AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -229,7 +241,9 @@ - (void)displayWillStart
       if (result) {
         self.image = result;
         _imageLoaded = YES;
-        _currentImageQuality = 1.0;
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Because it is (apparently?) important to have this dispatch even if already running on main, please add a comment here about why that is.  This is key because a later author could easily refactor it to be an ASPerformBlockOnMainThread() otherwise.